### DISCUSSION TITLE
Help Required: How can I convert selected slide  as image in power point js api ?

### DISCUSSION THREAD

**Discussion by ergauravgoyal** [OfficeDev/office-js#4061] [Q&A]
I have a usecase to convert selected slide to into base64 image in power point js api ?

**Comment by vegarw** [comment]
This is not directly supported unfortunately. It appears that the only current way to achieve this is to get the full presentation contents (supported in Office.js) and pass the contents to a backend service that can process the PowerPoint into images. For large presentations this becomes painfully slow, even just getting the contents of the presentation can take seconds. Hopefully there are others with the same need, so I will create a feature request in the appropriate forum and post back here.
